papis/commands/cache.py

```diff
@@ -0,0 +1,213 @@
+"""
+This command allows the user to interact to the papis cache or papis database.
+
+Before ``v0.14``, you could only clear the cache by doing
+
+::
+
+    papis --clear-cache
+
+Now you can do the same by typing the following command
```

Comment:
```suggestion
To clear the cache (remove it from the filesystem), you can run the following
command
```
? It's probably just confusing to mention the old way of doing it (that gets removed anyway)?

Reply:
agreed!